docs/node/migrations/zero-downtime-migrations/migration-proposal.mdx

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 1
+sidebar_label: "Migration Proposal"
+id: migration-proposal
+title: Migration Proposal
+description: "Migration Proposal"
+slug: /node/migrations/zero-downtime-migrations/migration-proposal
+---
+
+Migration Declaration is a network governance proposal to coordinate the network migration process. The migration proposal includes the activation period, duration, and the new chain ID that the network will migrate to.
```

Comment:
Similar to the Offline migration section, I also think you need something to introduce that we are now talking about zero-downtime migrations. For example:

```
Zero-downtime migrations begin with validators proposing, voting, and agreeing for a migration to take place. Anyone (validator, accounts, etc.) can submit a migration proposal; however, only validators can vote on and initiate the validation process.
```

Reply:
Would also be good here to list the stages of migration: proposal, approval, activation delay period (?), creation of new genesis state, streaming of change sets from old-> new network, recording of new changesets on new network in governance transactions, approval and application of changesets on net network, end of migration period.

Reply:
For the current paragraph though, some suggested rewording:

> A network migration is proposed by a validator using a special network
governance resolution.  The migration proposal includes the activation period,
duration, and the chain ID of the new network. The body of the migration
resolution comprises the fields of the following `MigrationDeclaration` struct.